platform/net/interface_configuration_creator_test.go

```diff
@@ -151,6 +151,34 @@ func describeInterfaceConfigurationCreator() {
 					})
 				})
 			})
+
+			Context("And the network has a alias", func() {
+				BeforeEach(func() {
+					staticNetwork.Alias = "static-interface-name"
```

Comment:
change "static-interface-name" => "static-interface-alias" or similar so it's clear from name